docs/UserGuide.md

```diff
@@ -233,82 +252,75 @@ Examples:
 
 - `findtag friends` - returns all valid contacts that are associated with tag being searched like `friends`, `Dewy Thompson`
 
-### Finding a contact tag: `findlead`
+### Finding a contact based on lead status: `findlead`
 
 Search for a contact based on their leads.
 
-Format: `findlead [STATUS]`
+Format: `findlead STATUS`
 
 The user can use either the long form or short form method to search
 
-The 4 stages of lead status
-
-- UNCONTACTED: `Uncontacted` or `U`
-- WORKING: `Working` or `W`
-- QUALIFIED: `Qualified` or `Q`
-- UNQUALIFIED: `Unqualified` or `X`
+| Type        | Formats (Case-sensitive) | Meaning                                                                                                                                                      |
+| ----------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| UNCONTACTED | `Uncontacted` or `U`     | The user has not gotten in touch with (contacted) the saved person. By default, newly added contacts have this status                                        |
+| WORKING     | `Working` or `W`         | The person has been contacted. The user is currently nurturing a relationship with the contact with the hopes of making them a qualified lead.               |
+| QUALIFIED   | `Qualified` or `Q`       | The contact is a client that has been nurtured to a ready, buying customer. A contact that has a prospect to buy or is in the sales funnel should go here.   |
+| UNQUALIFIED | `Unqualified` or `X`     | This status should only be used when it is certain that the customer's intents are not a match for the user's sales, and have no prospects of buying at all. |
 
-- The search is case-insensitive. e.g `u` will match `U`
 - The search works for both short form and long form. e.g `U` will match `Uncontacted`
 - Persons matching the tag will be returned
   e.g. `findlead U` will return `Hans Gruber` who is a contact with a lead status of `Uncontacted`
 
 Examples:
 
-- `findlead [STATUS]` - returns all contacts with the associated lead status, `Dewy Thompson` or `Majorie Dewy`
+- `findlead STATUS` - returns all contacts with the associated lead status, `Dewy Thompson` or `Majorie Dewy`
 
-### Finding a contact tag: `findall`
+### Finding a contact based on keyword: `findall`
 
-Search for a contact based on all attributes of a Person recorded in the database.
+Search for a contact based on matching KEYWORD to any attribute of a Person recorded in the contact list.
 
-Format: `findall [ATTRIBUTE]`
+Format: `findall KEYWORD`
 
-*Findall does not for parse through `[TAG]` and `[STATUS]` attributes as they have their own seperate commands for searching
+\*Does not for parse through `TAG` and `STATUS` attributes as they have their own seperate commands for searching
+\*Does not parse through tasklist and transactions (For future extensions)
```

Comment:
Would a line break be better for these 2 lines? Currently they are on a single line.